src/main/java/ch/epfl/biop/wrappers/cellpose/CellposeTaskSettings.java

```diff
@@ -59,23 +58,53 @@ public CellposeTaskSettings setFlowTh(double flow_threshold) {
         return this;
     }
 
-    /*
+    public CellposeTaskSettings setCellProbTh(double cellprob_threshold) {
+        this.cellprob_threshold = cellprob_threshold;
+        return this;
+    }
+
     public CellposeTaskSettings setAnisotropy(double anisotropy) {
         this.anisotropy = anisotropy;
         return this;
     }
-    */
-    public CellposeTaskSettings setCellProbTh(double cellprob_threshold) {
-        this.cellprob_threshold = cellprob_threshold;
+
+    public CellposeTaskSettings setStitchThreshold(double stitch_threshold){
+        this.stitch_threshold = stitch_threshold;
         return this;
     }
 
+    public CellposeTaskSettings setDo3D() {
+        this.use3D = true;
+        return this;
+    }
+
+    public CellposeTaskSettings do3D(boolean do3D) {
+        this.use3D = do3D;
+        return this;
+    }
+
+    public CellposeTaskSettings setOmni(boolean omni) {
+        this.omni = omni;
+        return this;
+    }
+
+    public CellposeTaskSettings setCluster(boolean cluster) {
+        this.cluster = cluster;
+        return this;
+    }
     public CellposeTaskSettings setFromPrefs() {
         String keyPrefix = Cellpose.class.getName() + ".";
         this.useGpu = Prefs.get(keyPrefix + "useGpu", Cellpose.useGpu);
         this.useMxnet = Prefs.get(keyPrefix + "useMxnet", Cellpose.useMxnet);
         this.useFastMode = Prefs.get(keyPrefix + "useFastMode", Cellpose.useFastMode);
         this.useResample = Prefs.get(keyPrefix + "useResample", Cellpose.useResample);
+        this.version = Prefs.get(keyPrefix + "Version", Cellpose.version);
```

Comment:
Make a null check ? Or is Cellpose.version the default value in case the key does not exist ?

Reply:
Not sure I get why for this one and not for others?